content/reference/user/simulations/index.md

```diff
@@ -15,13 +15,13 @@ The Simulations view contains all the simulations you have configured and the re
 
```

Comment:
create-simulation3.png =>
Please:
 * no error in screenshot
 * show 3 differents pools (not the dropdown)